### DISCUSSION TITLE
How do you STOP playing?

### DISCUSSION THREAD

**Discussion by starman2k01** [music-assistant/support#2162] [Q&A] [answered]
Music Assistant is great, love it, works well. I'm trying to enable a start/stop music button in my home. I can start playing an artist, but how can I remotely STOP it? There is no "stop playing" function.
Also, my dashboard lovelace can't add music assistant as a media player! It's not an entity I can display, as far as I can tell.
Help?

**Comment by OzGav** [comment]
https://www.home-assistant.io/integrations/media_player/ Use the media_stop service call. 

If you have added the integration then the MA media players will be available

**Reply by starman2k01** [reply]
Yes, I did use that to stop playing on the entities that were playing. The disadvantage is I have 30 speaker entities and many zones; I have to know exactly what speakers are playing to stop them. If I had a call directly to MA to stop music, I wouldn't have to know what speakers are playing.

**Reply by OzGav** [reply, marked as answer]
You can’t do that in MA either as each queue is unique. You could use a script

**Reply by huseman21** [reply]
Thats great if your a programmer and know  everything about everything.   How about just  making it stop when we say stop,
